src/main/java/org/cyclonedx/generators/json/BomJsonGenerator12.java

```diff
@@ -38,7 +41,17 @@
      * @param bom the BOM to generate
      */
     public BomJsonGenerator12(final Bom bom) {
-        this.bom = bom;
+        Bom modifiedBom = null;
+        try {
+            modifiedBom = injectBomFormatAndSpecVersion(bom);
+        }
+        catch (GeneratorException e) {
```

Comment:
Should we do something with this exception?

Reply:
If there were a logger, it would be a good spot to log something.